app/test/frontend/golden/flutter_packages_index_page2.html

```diff
@@ -114,20 +114,22 @@
 
 <div class="pagination">
   <ul>
-      <li class="">
-        <a href="/packages?page=1">
+      <li class=" ">
+        <a href="&#47;packages?page=1">
```

Comment:
This seems broken...

Reply:
We have `&` in these URLs, and that needs to be escaped. Unfortunately the HtmlEscaper is a bit too eager to do other escaping, but it is valid XML attribute, and the browser treats it OK.

Reply:
Switched to an attribute-specific HTML escaper, this is now better.